src/components/entries/Simple.js

```diff
@@ -0,0 +1,57 @@
+import {
+  useMemo
+} from 'preact/hooks';
+
+/**
+ * @param {Object} props
+ * @param {Boolean} props.debounce
+ * @param {Boolean} props.disabled
+ * @param {Object} props.element
+ * @param {Function} props.getValue
+ * @param {String} props.id
+ * @param {Function} props.setValue
+ */
+export default function Simple(props) {
+  const {
+    element,
+    id,
+    debounce,
+    disabled,
+    getValue,
+    setValue
+  } = props;
```

Comment:
Why don't we get `onFocus` and `onBlur` explicitly here?

Reply:
Good catch, there is no reason for that. I will update 👍